src/actions/user.actions.js

```diff
@@ -152,11 +154,33 @@ const resetPasswordConfirm = (password, passwordConfirm, urlUid, urlToken) => {
   };
 };
 
+const getTargets = () => {
+  return async (dispatch) => {
+    const { data: targets } = await targetService.getTargets();
+    dispatch({
+      type: GET_TARGETS_SUCCESS,
+      result: targets,
+    });
+  };
+};
+
+const getMatches = () => {
```

Comment:
```suggestion
const getMatches = async (dispatch) => {
    const { data: matches } = await targetService.getMatches();
    dispatch({
      type: GET_MATCHES_SUCCESS,
      result: matches,
    });
  };
```
Same for _getTargets_